modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/config/DefaultImplicitConfigurationProviderFactory.java

```diff
@@ -37,10 +36,9 @@
    */
   @Override
   public ConfigurationProvider createImplicitConfigurationProvider(ExtensionModel extensionModel,
+                                                                   ConfigurationModel implicitConfigurationModel,
                                                                    Event event,
                                                                    MuleContext muleContext) {
-    ConfigurationModel implicitConfigurationModel = getFirstImplicit(extensionModel.getConfigurationModels());
-
     if (implicitConfigurationModel == null) {
```

Comment:
should also check that the model is in fact implicit. The method in the ExtnesionManager which checks that should be given a better name and moved into a utils class. Then, reuse it here.